go.mod

```diff
@@ -4,7 +4,10 @@ go 1.21
 
 replace cloud.google.com/go => cloud.google.com/go v0.102.1
 
-replace github.com/vmware-tanzu/tanzu-cli/test/e2e/framework => ./test/e2e/framework
+replace (
+	github.com/vmware-tanzu/tanzu-cli/test/e2e/framework => ./test/e2e/framework
+	github.com/vmware-tanzu/tanzu-plugin-runtime => github.com/anujc25/tanzu-plugin-runtime v0.0.0-20231012181610-cb6d0aaa29e8
```

Comment:
This needs to be updated once https://github.com/vmware-tanzu/tanzu-plugin-runtime/pull/112 gets merged.